windows_containers/windows-containers-release-notes-7-x.adoc

```diff
@@ -28,10 +28,20 @@ For more information, see the Red Hat OpenShift Container Platform Life Cycle Po
 If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support.
 endif::openshift-origin[]
 
+[id="wmco-7-0-1"]
+== Release notes for Red Hat Windows Machine Config Operator 7.0.1
+
+This release of the WMCO provides new features and bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 7.0.1 were released in link:https://access.redhat.com/errata/RHBA-2023:0748[RHBA-2023:0748].
+
+[id="wmco-7-0-1-bug-fixes"]
+=== Bug fixes
+
+* Previously, WMCO 7.0.0 did not support running in a namespace other than `openshift-windows-machine-operator`. With this fix, you can run WMCO in a custom namespace. (link:https://issues.redhat.com/browse/OCPBUGS-5065[**OCPBUGS-5065**])
```

Comment:
Could we also mention the upgrade with custom namespace? "as well as perform upgrades having WMCO installed in a custom namespace"